docs/ensureSymlink-sync.md

```diff
@@ -1,20 +1,21 @@
-# ensureSymlinkSync(srcpath, dstpath[, type])
+# ensureSymlinkSync(srcPath, destPath[, type])
 
 Ensures that the symlink exists. If the directory structure does not exist, it is created.
 
 **Alias:** `createSymlinkSync()`
 
-- `srcpath` `<String>`
-- `dstpath` `<String>`
+- `srcPath` `<String>`
+- `destPath` `<String>`
 - `type` `<String>`
+	- It is only available on Windows and ignored on other platforms. It can be set to `dir`, `file`, or `junction`.
```

Comment:
```suggestion
- `type` `<String>` It is only available on Windows and ignored on other platforms. It can be set to `dir`, `file`, or `junction`.
```